OTRS - use one sessionId instead of creating a new one (#9506)
demisto/content: Packs/OTRS/ReleaseNotes/1_0_2.md

#### Integrations
##### OTRS

> - Fixed an issue where the integration created a new session for every time command executed.

Reviewer: ```suggestion
- Fixed an issue where the integration created a new session each time a command was executed.
```
